pgvector-14.yaml

```diff
@@ -1,5 +1,5 @@
 package:
```

Comment:
Similar to pgaudit, can we add a comment to each of these melanges summarising? i.e from pgaudit:

```
# pgAudit v1.6.X is intended to support PostgreSQL 14.
# https://github.com/pgaudit/pgaudit/blob/master/README.md#postgresql-version-compatibility
```

Reply:
added.